functions/changes-json/index.js

```diff
@@ -0,0 +1,33 @@
+require("babel-polyfill");
+
+const {
+  BinarySplitter,
+  sinks: { Kinesis },
+  sources: { Changes }
+} = require("osm-replication-streams");
+const osm2obj = require('osm2obj');
+const stringify = require("stringify-stream");
+
+let checkpointStream = require('./lib/checkpoint-stream');
+
+exports.handle = (event, context, callback) => {
+  checkpointStream(Changes, (err, stream) => {
```

Comment:
Can you either `return` immediately or remove the wrapping braces since it's a single statement? (Local style.) (Ditto elsewhere.)